operators/derive/to_mapbox.py

```diff
@@ -0,0 +1,99 @@
+import dataflows as DF
+from dataflows_airtable import load_from_airtable
+
+from conf import settings
+from operators.mapbox_upload import upload_tileset
+from srm_tools.logger import logger
+
+from . import helpers
+
+
+def geo_data_flow():
+    return DF.Flow(
+        DF.load(f'{settings.DATA_DUMP_DIR}/table_data/datapackage.json'),
+        DF.update_package(name='Geo Data'),
+        DF.update_resource(['table_data'], name='geo_data', path='geo_data.csv'),
+        DF.join_with_self(
+            'geo_data',
+            ['response_id', 'branch_geometry'],
+            fields=dict(
+                branch_geometry=None,
+                response_id=None,
+                response_name=None,
+                organization_id={'name': 'organization_id', 'aggregate': 'set'},
```

Comment:
Doing aggregates like this is not very useful - as we won't be able to match the different values of each branch.
It would be best to collect all the fields into a single object, and aggregate that instead into a list.

Reply:
Yes.

Reply:
yes, I see :)